api/resources/translations/messages-ne.properties

```diff
@@ -44,24 +51,24 @@ Confirm = पुष्टी गर्नुहोस्
 Confirm\ Password = पासवर्ड पृष्टि गर्नु
 Confirm\ delete = के {{name}} मेटाउन तपाई निश्चित हुनुहुन्छ? मेटिएको रेकर्ड फेरी ल्याउन मिल्दैन
 Contact = सम्पर्क व्यक्ति
-Contact\ type = 
+Contact\ type = सम्पर्क प्रकार
 Contacts = सम्पर्क
 Contacts\ file\ help = सम्पर्क फाइल सहयोग
 Content = बिषय सुची
 Continue = सुचारु
-Current\ Password = 
+Current\ Password = अहिलेको पासवर्ड
 Dashboard\ settings\ page = ड्यासबोर्ड सेटिङ्ग पृष्ठ
 Date\ display\ format = मिति देखाउने ढाचा
 Datetime\ display\ format = मिति सम्म देखाउने ढाचा
-Default = पुर्बनिर्धरित
+Default = पुर्वनिर्धारित
 Default\ Application\ Language = यो एप्लिकेशनको लागि पुर्वनिर्धारित भाषा
-Default\ country\ code = पुर्बनिर्धारित राष्ट्र कोड
-Default\ country\ code\ help = यो कोड देश कोड बिना सिस्टम मा प्रविष्टि सबै फोन नम्बर ठहराया गरिनेछ ।
+Default\ country\ code = पुर्वनिर्धारित राष्ट्र कोड
+Default\ country\ code\ help = देशको कोड बिना राखिएका सबै नम्बर हरुमा यो कोड थपिने छ। 
 Delete = मेटाउनुहोस्
 Deleting = मेटाइरहेको
-Disable = अयोग्य बनाउनुहोस
-Discard\ changes\ to\ current\ language = तपाईं जारी र हालको भाषा गरे परिवर्तनहरू रद्द गर्न चाहनुहुन्छ ?
-Display = 
+Disable = डिसेबल गर्नुहोस्
+Discard\ changes\ to\ current\ language = यो भाषामा गरेका परिवर्तन हरु सेभ नगरिकन अगाडी बढ्न चाहनु हुन्छ?
```

Comment:
परिवर्तन हरु to परिवर्तनहरु? extra space

Reply:
It should be हरू everywhere, not हरु.  (रू vs रु)

Reply:
I think we don't actually pluralize as frequently in Nepali as we pluralize nouns in English.